swig/matlab/helicsStartup.m

```diff
@@ -0,0 +1 @@
+loadlibrary(GetFullPath('./libhelicsSharedLib.dylib'));
```

Comment:
this seems mac specific,  is this going to be useful in general or just on a mac?

Reply:
Ah shoot you are right. That is Mac specific. I'll fix it now.

Reply:
might need to be a cmake configure file

Reply:
I don't seem to need that command on windows, it mostly just finds it if it is in the same directory.  Is there something we could put in the rpath to not need this on other platforms